cmd/wallet-web/src/pages/Store.vue

```diff
@@ -248,6 +247,8 @@ export default {
     fetchAllVaults: async function (token, collectionManager) {
       const { contents } = await collectionManager.getAll(token);
       this.vaults = Object.values(contents).map((vault) => vault);
+      // Default vault is selected vault by default, it is created on vault setup and must be only one.
+      this.setSelectedVault = this.vaults.find((vault) => vault.name === 'Default Vault').id;
```

Comment:
You didn't reuse the `setSelectedVault` method to update `this.selectedVault`

Reply:
updated